swayfmt/src/utils/map/byte_span.rs

```diff
@@ -44,7 +44,7 @@ impl ByteSpan {
     /// by a given Range<usize>, where 'start' is inclusive. This is to account for the start
     /// of the file.
```

Comment:
Maybe we can also mention the end is inclusive too in this doc comment, since stating start is inclusive explicitly feels like end is not?

Reply:
good catch, thank you!